doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,36 @@
+# QoS policies
+
+Unlike other middlewares like for example [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html)
+iceoryx keeps things simple and offers only basic Quality of Service (QoS) settings.The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings are similar to the respective DDS setting:
+
+| DDS                       | iceoryx                             | Explanation                                                                                                                    |
+|---------------------------|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| History depth of writer   | `PublisherOptions::historyCapacity` | This enables late-joining subscribers to request the n last samples                                                            |
+| History depth of reader   | `SubscriberOptions::queueCapacity`  | Size of the subscriber queue where samples are stored before they are passed to the user                                       |
+| -                         | `SubscriberOptions::historyRequest` | The number of samples a late-joining subscriber will request from a publisher, must be equal or smaller than `historyCapacity` |
```

Comment:
Why the comparison to DDS and not e.g. AUTOSAR? This is totally unrelated to iceoryx and confuses more than it helps. Please keep only the iceoryx related settings

Reply:
@elBoberido AUTOSAR has nothing that could be called similar. But I'm with you, as things are slightly different anyway, we can focus on iceoryx. The QoS mapping from iceoryx to DDS is a non-trivial topic that can be discussed when iceoryx is combined with a DDS stack like Cyclone DDS. Also the functionality we provide with iceoryx often ends up in a combination of history and durability QoS in DDS. This would make it even more complex to describe  the right mapping here

Reply:
Ok, I agree. Comparing can be even dangerous if it is easily misunderstood. Removed the DDS parts.